README.md

```diff
@@ -89,11 +89,27 @@ Certificates and configuration files that users will need are placed in the `con
 
 ### Apple Devices
 
-**Send users their Apple Profile.** Find the corresponding mobileconfig (Apple Profile) for each user and send it to them over AirDrop or other secure means. Apple Configuration Profiles are all-in-one configuration files for iOS and macOS devices. On macOS, double-clicking a profile to install it will fully configure the VPN. On iOS, users are prompted to install the profile as soon as the AirDrop is accepted.
+Apple devices can connect to an Algo VPN via IPsec using their built-in IPsec support or via WireGuard by installing WireGuard client software.
```

Comment:
--> by installing the [WireGuard app](https://itunes.apple.com/us/app/wireguard/id1441195209?ls=1&mt=8)

Reply:
The reason I used "client software" instead of "app" here is because this statement covers both iOS and macOS, and I didn't think the macOS app was ready yet.

Thanks for reviewing this PR. And for the iOS app!

Reply:
Ooo didn't realize that text was for macOS too. Indeed the macOS app isn't quite ready yet, but we're building it out of the same codebase as the iOS one, so things should wind up nicely integrated.